egui_web/src/http.rs

```diff
@@ -24,6 +24,10 @@ async fn fetch_jsvalue(request: &Request) -> Result<Response, JsValue> {
     opts.method(method);
     opts.mode(web_sys::RequestMode::Cors);
 
+    if method == &"POST".to_string() {
+        opts.body(Some(&JsValue::from_serde(body).unwrap()));
```

Comment:
`JsValue::from_serde` seems wrong. `JsValue::from` or `JsValue::from_string` or `from_str` or something.

Reply:
Yes I've only seen that after the first commit.